use_case_examples/hybrid_model/README.md

```diff
@@ -0,0 +1,14 @@
+# Hybrid model
+
+This use case example showcases how to partially run layers in FHE.
+
+In this case we apply a fully connected layer of a GPT-2 model in FHE.
+
+## How to run this use-case
+
+0. Install additional requirements using `python -m pip install -r requirements.txt`
+1. Compile GPT-2 model using `bash compile.sh` script
+1. Run FHE server using `bash serve.sh`
```

Comment:
server.sh ?

Reply:
No no, `serve.sh` as in to serve a FHE model.

Reply:
Got it thanks.